### PR TITLE
[JBPM-10218] Add italian translation

### PR DESCRIPTION
**Pull request by gjed**
Add support for italian translation

**JIRA**: [JBPM-10218](https://issues.redhat.com/browse/JBPM-10218)

**referenced Pull Requests**: _(please edit the URLs of referenced pullrequests if they exist)_

* [appformer#1413](https://github.com/kiegroup/appformer/pull/1413)
* [drools-wb#1573](https://github.com/kiegroup/drools-wb/pull/1573)
* [kie-wb-common#3822](https://github.com/kiegroup/kie-wb-common/pull/3822)
* [kie-wb-distributions#1239](https://github.com/kiegroup/kie-wb-distributions/pull/1239)

Any feedback is welcomed!

---
<details>
<summary>
How to replicate CI configuration locally?
</summary>

Build Chain tool does "simple" maven build(s), the builds are just Maven commands, but because the repositories relates and depends on each other and any change in API or class method could affect several of those repositories there is a need to use [build-chain tool](https://github.com/kiegroup/github-action-build-chain) to handle cross repository builds and be sure that we always use latest version of the code for each repository.
 
[build-chain tool](https://github.com/kiegroup/github-action-build-chain) is a build tool which can be used on command line locally or in Github Actions workflow(s), in case you need to change multiple repositories and send multiple dependent pull requests related with a change you can easily reproduce the same build by executing it on Github hosted environment or locally in your development environment. See [local execution](https://github.com/kiegroup/github-action-build-chain#local-execution) details to get more information about it.
</details>

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
 
* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>
  
* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

<details>
<summary>
How to backport a pull request to a different branch?
</summary>

In order to automatically create a **backporting pull request** please add one or more labels having the following format `backport-<branch-name>`, where `<branch-name>` is the name of the branch where the pull request must be backported to (e.g., `backport-7.67.x` to backport the original PR to the `7.67.x` branch).

> **NOTE**: **backporting** is an action aiming to move a change (usually a commit) from a branch (usually the main one) to another one, which is generally referring to a still maintained release branch. Keeping it simple: it is about to move a specific change or a set of them from one branch to another.

Once the original pull request is successfully merged, the automated action will create one backporting pull request per each label (with the previous format) that has been added.

If something goes wrong, the author will be notified and at this point a manual backporting is needed.

> **NOTE**: this automated backporting is triggered whenever a pull request on `main` branch is labeled or closed, but both conditions must be satisfied to get the new PR created.
</details>

### CODE REVIEW

**Comment by gjed** [comment]
Rebased gjed/i18n_it on main branch

**Comment by mareknovotny** [comment]
ok to test

File: jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants_it.properties [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+Release=Rilascia
+Claim=Richiedi
+ClaimAndWork=Richiedi e Lavora
+Work=Lavora
+Suspend=Sospendi
+Resume=Riprendi
+Details=Dettagli
+Actions=Azioni
+User=Utente
+Process_Id=Id Processo
```
Comment (pibizza) [inline jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants_it.properties:32]:
```suggestion
Process_Id=Id del Processo
```

File: jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants_it.properties [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+Process_Name=Nome Processo
+Process_Instance_Id=Id Istanza Processo
```
Comment (pibizza) [inline jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants_it.properties:34]:
```suggestion
Process_Instance_Id=Id dell''Istanza di Processo
```

File: jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants_it.properties [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+Process_Definition_Id=Id Definizione Processo
+Process_Instance_Details=Dettagli Istanza Processo
```
Comment (pibizza) [inline jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants_it.properties:36]:
```suggestion
Process_Instance_Details=Dettagli dell'Istanza di Processo
```

File: jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants_it.properties [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+Process_Instance_Correlation_Key=Chiave di Correlazione Istanza Processo
```
Comment (pibizza) [inline jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants_it.properties:37]:
```suggestion
Process_Instance_Correlation_Key=Chiave di Correlazione dell''Istanza di Processo
```

File: jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants_it.properties [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+Error_Count=Conteggio Errori
+Last_Modification_Date=Ultima Modifica
+No_Comments_For_This_Task=Non ci sono Commenti per questa attività
+Comment=Commento
+Added_At=Aggiunto il
+Added_By=Aggiunto Da
+Add_Comment=Aggiungi Commento
+Task_Must_Have_A_Name=L''Attività deve avere un nome
+Create=Crea
+Task_Name=Nome Attività
```
Comment (pibizza) [inline jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants_it.properties:48]:
```suggestion
Task_Name=Nome dell''Attività
```

File: jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants_it.properties [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+Description=Descrizione
+Comments=Commenti
+Process_Context=Contesto Processo
```
Comment (pibizza) [inline jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants_it.properties:51]:
```suggestion
Process_Context=Contesto del Processo
```

File: jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants_it.properties [hunk 7]
```diff
@@ -0,0 +1,170 @@
...
+ReleaseNotAllowedOn=Rilascio Attività ''{1}'' (id={0}) non consentito, l''attività non è Riservata o In Corso
+UnableToReleaseTask=Impossibile rilasciare l''attività ''{1}'' (Id={0}) errore: ''{2}''
+Bulk_Resume=Ripresa Multipla
+Resume_Confirmation=Sei sicuro di voler riprendere le attività selezionate?
+Resume_tasks=Riprendi attività selezionate
+ResumeNotAllowedOn=Ripresa Attività ''{1}'' (id={0}) non consentita, l''attività non è in stato Sospesa
+UnableToResumeTask=Impossibile riprendere l''attività ''{1}'' (Id={0}) errore: ''{2}''
+Bulk_Suspend=Sospensione Multipla
+Suspend_Confirmation=Sei sicuro di voler sospendere le attività selezionate?
+Suspend_tasks=Sospendi attività selezionate
```
Comment (pibizza) [inline jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants_it.properties:154]:
```suggestion
Suspend_tasks=Sospendi le attività selezionate
```

File: jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants_it.properties [hunk 8]
```diff
@@ -0,0 +1,170 @@
...
+SortByDateDesc=Ordina per data discendente
+SortByDateAsc=Ordina per data ascendente
+Bulk_Claim=Richiesta Multipla
+Claim_Confirmation=Sei sicuro di voler richiedere le attività selezionate?
+Claim_tasks=Richiedi attività selezionate
+ClaimNotAllowedOn=Richiesta Attività ''{1}'' (id={0}) non consentita, l''attività non è nello stato Pronta
+UnableToClaimTask=Impossibile richiedere l''attività ''{1}'' (Id={0}) errore: ''{2}''
+Bulk_Release=Rilascio Multiplo
+Release_Confirmation=Sei sicuro di voler rilasciare le attività selezionate?
+Release_tasks=Rilascia attività selezionate
```
Comment (pibizza) [inline jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants_it.properties:144]:
```suggestion
Release_tasks=Rilascia le attività selezionate
```

File: jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants_it.properties [hunk 9]
```diff
@@ -0,0 +1,170 @@
...
+Completed=Completata
+Initiator=Iniziatore
+Task_Form=Modulo Attività
+Task_Form_Name=Nome Modulo Attività
+Task_Form_DeploymentId=Id Deploy Modulo Attività
+New_FilteredList=Nuova lista filtrata
+UnexpectedError=Errore imprevisto\: {0}
+ActivationTime=Tempo di Attivazione
+CreatedBy=Creato Da
+DeploymentId=Id Deploy
```
Comment (pibizza) [inline jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants_it.properties:85]:
```suggestion
DeploymentId=Id del Deploy
```

File: jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants_it.properties [hunk 10]
```diff
@@ -0,0 +1,170 @@
...
+Task_Admin=Amministratore
+FilterTaskAdmin=Filtro Amministratore
+DelegationUnable=Delega non valida
+High=Alta
+Medium=Media
+Low=Bassa
+Completed=Completata
+Initiator=Iniziatore
+Task_Form=Modulo Attività
+Task_Form_Name=Nome Modulo Attività
```
Comment (pibizza) [inline jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants_it.properties:79]:
```suggestion
Task_Form_Name=Nome del Modulo dell''Attività
```

File: jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants_it.properties [hunk 11]
```diff
@@ -0,0 +1,170 @@
...
+Provide_User_Or_Group=Fornisci almeno un utente o un gruppo
+Task_Admin=Amministratore
+FilterTaskAdmin=Filtro Amministratore
+DelegationUnable=Delega non valida
+High=Alta
+Medium=Media
+Low=Bassa
+Completed=Completata
+Initiator=Iniziatore
+Task_Form=Modulo Attività
```
Comment (pibizza) [inline jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants_it.properties:78]:
```suggestion
Task_Form=Modulo dell''Attività
```

File: jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants_it.properties [hunk 12]
```diff
@@ -0,0 +1,170 @@
...
+Task_Form_Name=Nome Modulo Attività
+Task_Form_DeploymentId=Id Deploy Modulo Attività
+New_FilteredList=Nuova lista filtrata
+UnexpectedError=Errore imprevisto\: {0}
+ActivationTime=Tempo di Attivazione
+CreatedBy=Creato Da
+DeploymentId=Id Deploy
+DueDate=Scadenza
+ParentId=Id Padre
+ProcessSessionId=Id Sessione Processo
```
Comment (pibizza) [inline jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants_it.properties:88]:
```suggestion
ProcessSessionId=Id della Sessione di Processo
```

File: jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants_it.properties (hunk 11 above)
Comment (pibizza) [inline jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants_it.properties:78]:
```suggestion
Task_Form=Modulo dell''Attività
```

File: jbpm-wb-integration/jbpm-wb-integration-client/src/main/resources/org/jbpm/workbench/wi/client/i18n/Constants_it.properties [hunk 13]
```diff
@@ -0,0 +1,105 @@
+No_Deployment_Units_Available=Nessuna Unità di Deploy Disponibile
+Deployment_Units=Unità di Deploy
+Undeploy=Annulla Deploy
```
Comment (pibizza) [inline jbpm-wb-integration/jbpm-wb-integration-client/src/main/resources/org/jbpm/workbench/wi/client/i18n/Constants_it.properties:3]:
```suggestion
Undeploy=Annulla il Deploy
```

File: jbpm-wb-integration/jbpm-wb-integration-client/src/main/resources/org/jbpm/workbench/wi/client/i18n/Constants_it.properties [hunk 14]
```diff
@@ -0,0 +1,105 @@
...
+Details=Dettagli
+Deploy_Unit=Deploy Unità
+Deploy_A_New_Unit=Deploy una Nuova Unità
+GroupID=ID Gruppo
```
Comment (pibizza) [inline jbpm-wb-integration/jbpm-wb-integration-client/src/main/resources/org/jbpm/workbench/wi/client/i18n/Constants_it.properties:7]:
```suggestion
GroupID=ID del Gruppo
```

File: jbpm-wb-integration/jbpm-wb-integration-client/src/main/resources/org/jbpm/workbench/wi/client/i18n/Constants_it.properties [hunk 15]
```diff
@@ -0,0 +1,105 @@
...
+Deployed_Units_Refreshed=Unità Deployate Aggiornate
+Unit=Unità
+Please_Wait=Attendere Prego
+KIE_Configurations=Configurazioni KIE
+New_Deployment_Unit=Nuova Unità di Deploy
+Strategy=Strategia di Runtime
+Deployment=Deploy
+Undeploy_Question=Sei sicuro di voler annullare il deploy di questa unità?
+DeploymentDescriptorEditor=Editor del Descrittore di Deploy
+PersistenceUnit=Nome Persistence unit
```
Comment (pibizza) [inline jbpm-wb-integration/jbpm-wb-integration-client/src/main/resources/org/jbpm/workbench/wi/client/i18n/Constants_it.properties:23]:
```suggestion
PersistenceUnit=Nome dell'Unità di Persistenza
```

File: jbpm-wb-integration/jbpm-wb-integration-client/src/main/resources/org/jbpm/workbench/wi/client/i18n/Constants_it.properties [hunk 16]
```diff
@@ -0,0 +1,105 @@
...
+AuditPersistenceUnit=Nome Persistence unit di Audit
```
Comment (pibizza) [inline jbpm-wb-integration/jbpm-wb-integration-client/src/main/resources/org/jbpm/workbench/wi/client/i18n/Constants_it.properties:24]:
```suggestion
AuditPersistenceUnit=Nome dell'Unità di Persistenza di Audit
```

File: jbpm-wb-integration/jbpm-wb-integration-client/src/main/resources/org/jbpm/workbench/wi/client/i18n/Constants_it.properties [hunk 17]
```diff
@@ -0,0 +1,105 @@
...
+EnvironmentEntries=Environment entries
+Configuration=Configurazione
+RequiredRoles=Ruoli Richiesti
+RemoteableClasses=Classi Remoteable
+LimitSerializationClasses=Limita Classi di Serializzazione
+NamedParams=Parametri nominati disponibili
+NoParamResolver=Il resolver selezionato non supporta i parametri
+Type=Tipo
+MergeMode=Modalità di Merge (descrittore di deploy)
+EnterValue=Inserisci valore
```
Comment (pibizza) [inline jbpm-wb-integration/jbpm-wb-integration-client/src/main/resources/org/jbpm/workbench/wi/client/i18n/Constants_it.properties:53]:
```suggestion
EnterValue=Inserisci il valore
```

File: jbpm-wb-integration/jbpm-wb-integration-client/src/main/resources/org/jbpm/workbench/wi/client/i18n/Constants_it.properties [hunk 18]
```diff
@@ -0,0 +1,105 @@
...
+InternalUploadError=Impossibile aggiungere l''attività personalizzata a causa di un errore interno
+AddTaskSuccess=Attività personalizzate aggiunte con successo -
+SkippedMessage=Le seguenti attività personalizzate sono state saltate in quanto il repository contiene già servizi con lo stesso nome: {0}.
+RemoveTaskSuccess=Attività personalizzata {0} rimossa con successo.
+Installing=Installazione dell''attività personalizzata in corso...
+InstallServiceTaskParams=Parametri richiesti per l''attività personalizzata
+RemoveServiceTaskAction=Elimina
+RemoveServiceTaskTitle=Elimina Attività Personalizzata
+RemoveServiceTaskConfirmMessage=Sei sicuro di voler rimuovere l''attività personalizzata {0} ?
+CanNotRemoveServiceTaskMessage=Impossibile rimuovere l''attività personalizzata {0}, in quanto è utilizzata da almeno un progetto. Rimuoverla da tutti i progetti prima.
```
Comment (pibizza) [inline jbpm-wb-integration/jbpm-wb-integration-client/src/main/resources/org/jbpm/workbench/wi/client/i18n/Constants_it.properties:104]:
```suggestion
CanNotRemoveServiceTaskMessage=Impossibile rimuovere l''attività personalizzata {0}, in quanto è utilizzata da almeno un progetto. Rimuoverla  prima da tutti i progetti.
```

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/resources/org/jbpm/workbench/pr/client/resources/i18n/Constants_it.properties [hunk 19]
```diff
@@ -0,0 +1,190 @@
...
+Previous_Value=Valore precedente
+No_History_For_This_Variable=Nessun storico disponibile per questa variabile
+ProcessDefinitionBreadcrumb=Definizione di processo: {0}
+Process_Instance_Details=Dettagli istanza
+Process_Definition_Version=Versione definizione
+Process_Instance_State=Stato istanza
+Current_Activities=Attività correnti
+Clear=Cancella
+Save=Salva
+Variables_Name=Nome variabile
```
Comment (pibizza) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/resources/org/jbpm/workbench/pr/client/resources/i18n/Constants_it.properties:56]:
```suggestion
Variables_Name=Nome della variabile
```

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/resources/org/jbpm/workbench/pr/client/resources/i18n/Constants_it.properties [hunk 20]
```diff
@@ -0,0 +1,190 @@
...
+Variable_Value=Valore variabile
```
Comment (pibizza) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/resources/org/jbpm/workbench/pr/client/resources/i18n/Constants_it.properties:57]:
```suggestion
Variable_Value=Valore della variabile
```

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/resources/org/jbpm/workbench/pr/client/resources/i18n/Constants_it.properties [hunk 21]
```diff
@@ -0,0 +1,190 @@
...
+Signaling_Process_Instance_Not_Allowed=Impossibile segnalare l''istanza del processo (L''istanza del processo non è attiva) (id={0})
+Bulk_Signal=Segnale di massa
+ProcessInstanceBreadcrumb=Istanza del processo: {0}
+History=Storico
+Edit=Modifica
+Last_Modification=Ultima modifica
+Value=Valore
+Type=Tipo
+No_Variables_Available=Nessuna variabile di processo disponibile
+View_Process_Instances=Visualizza istanze di processo
```
Comment (pibizza) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/resources/org/jbpm/workbench/pr/client/resources/i18n/Constants_it.properties:36]:
```suggestion
View_Process_Instances=Visualizza le istanze di processo
```

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/resources/org/jbpm/workbench/pr/client/resources/i18n/Constants_it.properties [hunk 22]
```diff
@@ -0,0 +1,190 @@
...
+New_Process_Instance=Nuova istanza di processo
+View_Process_Model=Modello di processo
+Process_Definition_Id=ID definizione
```
Comment (pibizza) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/resources/org/jbpm/workbench/pr/client/resources/i18n/Constants_it.properties:39]:
```suggestion
Process_Definition_Id=ID della definizione
```

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/resources/org/jbpm/workbench/pr/client/resources/i18n/Constants_it.properties [hunk 23]
```diff
@@ -0,0 +1,190 @@
...
+Process_Definition_Name=Nome definizione
```
Comment (pibizza) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/resources/org/jbpm/workbench/pr/client/resources/i18n/Constants_it.properties:40]:
```suggestion
Process_Definition_Name=Nome della definizione
```

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/resources/org/jbpm/workbench/pr/client/resources/i18n/Constants_it.properties [hunk 24]
```diff
@@ -0,0 +1,190 @@
...
+Deployment_Name=Deployment
+Human_Tasks=Attività umane
+User_And_Groups=Utenti e gruppi
+SubProcesses=Sottoprocessi
+Process_Variables=Variabili di processo
+Previous_Value=Valore precedente
+No_History_For_This_Variable=Nessun storico disponibile per questa variabile
+ProcessDefinitionBreadcrumb=Definizione di processo: {0}
+Process_Instance_Details=Dettagli istanza
+Process_Definition_Version=Versione definizione
```
Comment (pibizza) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/resources/org/jbpm/workbench/pr/client/resources/i18n/Constants_it.properties:51]:
```suggestion
Process_Definition_Version=Versione della definizione
```

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/resources/org/jbpm/workbench/pr/client/resources/i18n/Constants_it.properties [hunk 25]
```diff
@@ -0,0 +1,190 @@
...
+Human_Tasks_Count=Conteggio attività umane
+Deployment_Name=Deployment
+Human_Tasks=Attività umane
+User_And_Groups=Utenti e gruppi
+SubProcesses=Sottoprocessi
+Process_Variables=Variabili di processo
+Previous_Value=Valore precedente
+No_History_For_This_Variable=Nessun storico disponibile per questa variabile
+ProcessDefinitionBreadcrumb=Definizione di processo: {0}
+Process_Instance_Details=Dettagli istanza
```
Comment (pibizza) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/resources/org/jbpm/workbench/pr/client/resources/i18n/Constants_it.properties:50]:
```suggestion
Process_Instance_Details=Dettagli dell'istanza
```

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/resources/org/jbpm/workbench/pr/client/resources/i18n/Constants_it.properties [hunk 26]
```diff
@@ -0,0 +1,190 @@
...
+Process_Definition_Version=Versione definizione
+Process_Instance_State=Stato istanza
```
Comment (pibizza) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/resources/org/jbpm/workbench/pr/client/resources/i18n/Constants_it.properties:52]:
```suggestion
Process_Instance_State=Stato dell''istanza
```

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/resources/org/jbpm/workbench/pr/client/resources/i18n/Constants_it.properties [hunk 27]
```diff
@@ -0,0 +1,190 @@
...
+ErrorCountNumberView=Fai clic per visualizzare {0} errori
+ErrorCountNumberView[\=1]=Fai clic per visualizzare {0} errore
+FilterByProcessInstanceId=Filtro per ID istanza di processo...
+FilterByInitiator=Filtro per iniziatore...
+FilterByCorrelationKey=Filtro per chiave di correlazione...
+FilterByDescription=Filtro per descrizione...
+WithoutErrors=Senza errori
+WithErrors=Con errori
+ViewJobs=Visualizza lavori
+ViewTasks=Visualizza attività
```
Comment (pibizza) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/resources/org/jbpm/workbench/pr/client/resources/i18n/Constants_it.properties:112]:
```suggestion
ViewTasks=Visualizza le attività
```

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/resources/org/jbpm/workbench/pr/client/resources/i18n/Constants_it.properties [hunk 28]
```diff
@@ -0,0 +1,190 @@
...
+SelectServerTemplate=Seleziona configurazione del server
+ErrorCountNumberView=Fai clic per visualizzare {0} errori
+ErrorCountNumberView[\=1]=Fai clic per visualizzare {0} errore
+FilterByProcessInstanceId=Filtro per ID istanza di processo...
+FilterByInitiator=Filtro per iniziatore...
+FilterByCorrelationKey=Filtro per chiave di correlazione...
+FilterByDescription=Filtro per descrizione...
+WithoutErrors=Senza errori
+WithErrors=Con errori
+ViewJobs=Visualizza lavori
```
Comment (pibizza) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/resources/org/jbpm/workbench/pr/client/resources/i18n/Constants_it.properties:111]:
```suggestion
ViewJobs=Visualizza i lavori
```

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/resources/org/jbpm/workbench/pr/client/resources/i18n/Constants_it.properties [hunk 29]
```diff
@@ -0,0 +1,190 @@
...
+Owner=Proprietario
+Project=Progetto
+Loading=Caricamento...
+VariableValueUpdated=Variabile {0} aggiornata
+Ok=Ok
+Correlation_Key=Chiave di correlazione
+Last_Modification_Date=Ultimo aggiornamento
+Last_Modification_Date_Placeholder=Ultimo aggiornamento...
+Parent_Process_Instance=Istanza del processo padre
+Parent_Process_Instance_Id=ID istanza del processo padre
```
Comment (pibizza) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/resources/org/jbpm/workbench/pr/client/resources/i18n/Constants_it.properties:87]:
```suggestion
Parent_Process_Instance_Id=ID dell'istanza del processo padre
```

**Review by pibizza — CHANGES_REQUESTED** [review]
I have added a number of suggestions. Most of them are rather trivial to make the italian sound a bit more natural.

**Comment by mareknovotny** [comment]
it seems that i took suggestions wrongly as some had not escaped form or quotes :(

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/resources/org/jbpm/workbench/pr/client/resources/i18n/Constants_it.properties [hunk 30]
```diff
@@ -0,0 +1,190 @@
...
+Owner=Proprietario
+Project=Progetto
+Loading=Caricamento...
+VariableValueUpdated=Variabile {0} aggiornata
+Ok=Ok
+Correlation_Key=Chiave di correlazione
+Last_Modification_Date=Ultimo aggiornamento
+Last_Modification_Date_Placeholder=Ultimo aggiornamento...
+Parent_Process_Instance=Istanza del processo padre
+Parent_Process_Instance_Id=ID dell'istanza del processo padre
```
Comment (mareknovotny) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/resources/org/jbpm/workbench/pr/client/resources/i18n/Constants_it.properties:87]:
```suggestion
Parent_Process_Instance_Id=ID dell''istanza del processo padre
```

**Review by pibizza — APPROVED** [review]

File: jbpm-wb-forms/jbpm-wb-forms-client/src/main/resources/org/jbpm/workbench/forms/client/i18n/Constants_it.properties [hunk 31]
```diff
@@ -0,0 +1,31 @@
...
+TaskClaimed=Attività\: {0} richiesta\!
+TaskSaved=Attività\: {0} salvata\!
+TaskReleased=Attività\: {0} rilasciata\!
+UnableToFindFormForTask=Impossibile trovare un modulo per l''attività {0}. Riprova più tardi o contatta l''amministratore.
+Start_process_instance=Avvia istanza del processo
+Select_Process=Seleziona il processo da avviare
+Process_DeploymentId=Deployment del processo
+Process_Definition=Definizione del processo
+ProcessStarted=Processo Id: {0} avviato!
+CaseStarted=Caso con id {0} avviato!
```
[the remark this answers is not in the document]
Comment (mareknovotny) [reply jbpm-wb-forms/jbpm-wb-forms-client/src/main/resources/org/jbpm/workbench/forms/client/i18n/Constants_it.properties:27]: Isn't this better to keep id there? @pibizza ?

File: jbpm-wb-forms/jbpm-wb-forms-client/src/main/resources/org/jbpm/workbench/forms/client/i18n/Constants_it.properties [hunk 32]
```diff
@@ -0,0 +1,31 @@
...
+TaskStarted=Attività\: {0} avviata\!
+TaskClaimed=Attività\: {0} richiesta\!
+TaskSaved=Attività\: {0} salvata\!
+TaskReleased=Attività\: {0} rilasciata\!
+UnableToFindFormForTask=Impossibile trovare un modulo per l''attività {0}. Riprova più tardi o contatta l''amministratore.
+Start_process_instance=Avvia istanza del processo
+Select_Process=Seleziona il processo da avviare
+Process_DeploymentId=Deployment del processo
+Process_Definition=Definizione del processo
+ProcessStarted=Processo Id: {0} avviato!
```
[the remark this answers is not in the document]
Comment (mareknovotny) [reply jbpm-wb-forms/jbpm-wb-forms-client/src/main/resources/org/jbpm/workbench/forms/client/i18n/Constants_it.properties:26]: Isn't this better to keep id there? @pibizza ?

[the remark this answers is not in the document]
Comment (pibizza) [reply jbpm-wb-forms/jbpm-wb-forms-client/src/main/resources/org/jbpm/workbench/forms/client/i18n/Constants_it.properties:26]: We can keep it.

File: jbpm-wb-forms/jbpm-wb-forms-client/src/main/resources/org/jbpm/workbench/forms/client/i18n/Constants_it.properties (hunk 31 above)
[the remark this answers is not in the document]
Comment (pibizza) [reply jbpm-wb-forms/jbpm-wb-forms-client/src/main/resources/org/jbpm/workbench/forms/client/i18n/Constants_it.properties:27]: We can keep it

File: jbpm-wb-forms/jbpm-wb-forms-client/src/main/resources/org/jbpm/workbench/forms/client/i18n/Constants_it.properties (hunk 32 above)
[the remark this answers is not in the document]
Comment (mareknovotny) [reply jbpm-wb-forms/jbpm-wb-forms-client/src/main/resources/org/jbpm/workbench/forms/client/i18n/Constants_it.properties:26]: ok

**Pull request merged**